pkgs/development/python-modules/plum-py/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, buildPythonPackage, fetchFromGitLab, isPy3k, pytest, baseline }:
+
+buildPythonPackage rec {
+  pname = "plum-py";
+  version = "0.4.0";
+  disabled = !isPy3k;
+
+  src = fetchFromGitLab {
+    owner = "dangass";
+    repo = "plum";
+    rev = "6a9ff863c0e9fa21f7b2230d25402155a5522e4b";
+    sha256 = "1iv62yb704c61b0dvsmyp3j6xpbmay532g9ny4pw4zbg3l69vd5j";
+  };
+
+  postPatch = ''
+    substituteInPlace src/plum/int/flag/_flag.py \
+      --replace 'if sys.version_info < (3, 7):' 'if True:'
+  '';
+
+  checkInputs = [ pytest baseline ];
+  checkPhase = "pytest tests";
```

Comment:
```suggestion
  checkInputs = [ pytestCheckHook baseline ];
```

Reply:
https://github.com/NixOS/nixpkgs/pull/116570#discussion_r602584894

Reply:
Apologies, I think my previously reply got lost when I force-pushed again. Copying it here for reference:

pytestCheckHook (and plain "pytest") don't work in this repo for some reason. It looks like it doesn't set PYTHONPATH correctly:

```
_____________________ ERROR collecting tests/test_bytes.py _____________________
ImportError while importing test module '/build/source/tests/test_bytes.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/yl69v76azrz4daiqksrhb8nnmdiqdjg9-python3-3.8.8/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
E   ModuleNotFoundError: No module named 'tests.test_bytes'
```